eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala

```diff
@@ -220,19 +220,19 @@ object EventualCassandra {
     }
   }
 
-  final case class Statements[F[_]](
+  private[journal] final case class Statements[F[_]](
     records: JournalStatements.SelectRecords[F],
     metaJournal: MetaJournalStatements[F],
     selectOffset2: Pointer2Statements.SelectOffset[F],
   )
 
-  object Statements {
+  private[journal] object Statements {
```

Comment:
overall, it is wrong to mix several changes in single commit - in this PR we have:
* making some API private (without deprecation, though I am not sure, how easy it would be to deprecate externally while keeping it not-deprecated internally without changing the name)
* refactoring of existing code
* addition of documentation

IMHO it is better to create 3 PRs - each smaller and affecting single feature

Reply:
overall I agree but prefer to keep this PR as is mostly to avoid waisting time on splitting it